RecoLocalTracker/SiStripClusterizer/plugins/clusterGPU.cu

```diff
@@ -444,4 +840,336 @@ namespace stripgpu {
     }
 #endif
   }
+
+  void MkFitSiStripHitGPUKernel::loadBarrel(const std::vector<const GeometricDet*> dets_barrel, const std::vector<const GeomDet*> rots_barrel, const SiStripBackPlaneCorrection* BackPlaneCorrectionMap, const MagneticField* MagFieldMap,const SiStripLorentzAngle* LorentzAngleMap, const std::vector<std::tuple<unsigned int, float, float , float, float, float , float, float, float , float, float, float , float, float, float , float>> stripUnit){
+    printf("LOADING BARREL\n");
+    short* indexer3_h;
+    short* indexer5_h;
+    cudaMallocHost((void**)&indexer3_h,31214*sizeof(short));
+    cudaMallocHost((void**)&indexer5_h,46426*sizeof(short));
```

Comment:
I don't see anywhere these are freed?  Also, they should all be enclosed with cudaCheck().  Normally I'd suggest they also be converted to cms::cuda::make_host_unique(), but the geometry should as cudaMallocHost(), as this should turn into an ESProducer (which I'll do), and cms::cuda::make_host_unique() isn't supported yet in ESProducers.